AndroidSDK/src/com/leanplum/Leanplum.java

```diff
@@ -883,9 +886,28 @@ public void run() {
                 LeanplumInternal.onHasStartedAndRegisteredAsDeveloper();
               }
             }
-
             LeanplumInternal.moveToForeground();
             startHeartbeat();
+            // Send notification channels to the server on Android O and upper(only with target sdk
```

Comment:
should we extract this into a separate method?
the original method gets too long imo....